src/ucp/core/ucp_request.inl

```diff
@@ -318,6 +318,18 @@ ucp_request_try_send(ucp_request_t *req, unsigned pending_flags)
 static UCS_F_ALWAYS_INLINE void
 ucp_request_send(ucp_request_t *req, unsigned pending_flags)
 {
+    /* test all values used in ucp_request_send_state_ff */
+    if (req->send.state.uct_comp.func != NULL) {
```

Comment:
shall we put it under #if/#endif valgrind macro to make sure no extra code will be generated in release mode?

Reply:
nop - optimization compiler will exclude empty code

Reply:
👍